server/sandbox_run_linux.go

```diff
@@ -375,30 +376,30 @@ func (s *Server) runPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 		}
 	}()
 
-	if _, err := s.ReservePodName(sbox.ID(), sbox.Name()); err != nil {
-		reservedID, getErr := s.PodIDForName(sbox.Name())
+	if _, err := s.ReservePodName(sandboxID, sandboxName); err != nil {
+		reservedID, getErr := s.PodIDForName(sandboxName)
 		if getErr != nil {
-			return nil, fmt.Errorf("failed to get ID of pod with reserved name (%s), after failing to reserve name with %w: %w", sbox.Name(), getErr, getErr)
+			return nil, fmt.Errorf("failed to get ID of pod with reserved name (%s), after failing to reserve name with %w: %w", sandboxName, getErr, getErr)
 		}
 		// if we're able to find the sandbox, and it's created, this is actually a duplicate request
 		// Just return that sandbox
-		if reservedSbox := s.GetSandbox(reservedID); reservedSbox != nil && reservedSbox.Created() {
+		if reservedsbuilder := s.GetSandbox(reservedID); reservedsbuilder != nil && reservedsbuilder.Created() {
```

Comment:
this shouldn't be a builder this should be a sandbox, why the name change?

Reply:
it would reduce code churn too to start which would make this easier to review as well